### PR TITLE
[DFFS-4] Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by thebadmos**
Screenshot (14)

### CODE REVIEW

**Review by Ireti203 — CHANGES_REQUESTED** [review]
Weldone @thebadmos. But you are not doing what the task requested. You are not to create a separate file for navigation. You should add the navigation links to previously existing  pages. E,g if I click on "Products" on the Contact us page, it takes me to the Products page.

**Review by adaugochi — CHANGES_REQUESTED** [review]
Your `product.html` file should be inside your `html` folder not your `html-forms` folder

File: submissions/OlamijiBadmos/html-forms/contact.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
     <title>Dufuna-Fem | Contact Page</title>
 </head>
 <body>
+    <h4>The products in the store are  <a href="product.html">Product Page</a> </h4>
```
Comment (Ireti203) [inline submissions/OlamijiBadmos/html-forms/contact.html]: Include the link to the contact page too. Also, add the navigation links on the product.html file

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: submissions/OlamijiBadmos/html/product.html [hunk 2]
```diff
@@ -36,6 +38,7 @@ <h5>Similar Product Name</h5>
         
      </ol>
```
Comment (adaugochi) [inline submissions/OlamijiBadmos/html/product.html:39]: Remove newline spaces from line 40-42

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by adaugochi — APPROVED** [review]

**Pull request merged**